src/Knp/Component/Pager/Paginator.php

```diff
@@ -38,7 +38,10 @@ class Paginator implements PaginatorInterface
         self::SORT_DIRECTION_PARAMETER_NAME => 'direction',
         self::FILTER_FIELD_PARAMETER_NAME => 'filterParam',
         self::FILTER_VALUE_PARAMETER_NAME => 'filterValue',
-        self::DISTINCT => true
+        self::DISTINCT => true,
+        self::FORCE_VALID_PAGINATION_PARAMETERS => false,
+        self::DEFAULT_LIMIT => 10,
+        self::DEFAULT_PAGE_NUMBER => 1,
```

Comment:
@daria-sieroshtan 
I think it's better to have these two as separate constants, not in options